x/stakeibc/keeper/unbonding.go

```diff
@@ -485,8 +482,8 @@ func (k Keeper) UnbondFromHostZone(ctx sdk.Context, hostZone types.HostZone) err
 	}
 
 	// Send the messages in batches so the gas limit isn't exceedeed
```

Comment:
This comment is inaccurate right? Since we error if the number of undelegation messages is greater than the batch size

Reply:
good eye! updated